### PR TITLE
[-] BO : Fix #PSCSX-6663, bad table id after DND

### PR DESCRIPTION
**Pull request by gRoussac**
:tropical_drink:

### CODE REVIEW

File: js/admin/dnd.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ function initTableDnD(table)
 						id_module: ids[1],
 						way: way
 					};
-				else if (table.id.indexOf('attribute') != -1 && table.id != 'attribute_group') {
+				else if (table.id.indexOf('attribute') != -1 && tableId!= 'attribute_group') {
```
Comment (tchauviere) [inline js/admin/dnd.js]: Maybe we could could something more like:

```js
tableId != 'attribute'
```

Dont you think @gRoussac  ?

File: js/admin/dnd.js [hunk 2]
```diff
@@ -58,13 +58,13 @@ function initTableDnD(table)
 				var ids = row.id.split('_');
 				var tableDrag = table;
 				var params = '';
-				table.id = table.id.replace('table-', '');
-				if (table.id == 'cms_block_0' || table.id == 'cms_block_1')
+				tableId = table.id.replace('table-', '');
```
Comment (tchauviere) [inline js/admin/dnd.js]: Please add

```js
var
```

To avoid making it a global :) Thx

**Comment by gRoussac** [comment]
Sorry commit lost last week !

Done :tired_face:

**Comment by tchauviere** [comment]
:v:

**Pull request merged**